trunk/SUAVE/Methods/Weights/Correlations/FLOPS/prop_system.py

```diff
@@ -0,0 +1,227 @@
+            The Flight Optimization System Weight Estimation Method
+
+        Inputs:
+            vehicle - data dictionary with vehicle properties                   [dimensionless]
+                 -.design_mach_number: design mach number
+            prop    - data dictionary for the specific propulsor that is being estimated [dimensionless]
+                -.number_of_engines: number of engines
+                -.nacelle_diameter: diameter of nacelle                     [m]
+                -.sealevel_static_thrust: sealevel static thrust of engine  [N]
+
+        Outputs:
+            WEC: electrical engine control system weight                    [kg]
+            WSTART: starter engine weight                                   [kg]
+
+        Properties Used:
+            N/A
+    """
+    NENG = prop.number_of_engines
+    THRUST = prop.sealevel_static_thrust * 1 / Units.lbf
+    WEC = 0.26 * NENG * THRUST ** 0.5
+    FNAC = prop.nacelle_diameter / Units.ft
+    VMAX = vehicle.design_mach_number
+    WSTART = 11.0 * NENG * VMAX ** 0.32 * FNAC ** 1.6
+    return WEC * Units.lbs, WSTART * Units.lbs
+
+
+def fuel_system_FLOPS(vehicle, NENG):
+    """ Calculates the weight of the fuel system based on the FLOPS method
+        Assumptions:
+
+        Source:
+            The Flight Optimization System Weight Estimation Method
+
+        Inputs:
+            vehicle - data dictionary with vehicle properties                   [dimensionless]
+                -.design_mach_number: design mach number
+                -.mass_properties.max_zero_fuel: maximum zero fuel weight   [kg]
+
+        Outputs:
+            WFSYS: Fuel system weight                                       [kg]
+
+        Properties Used:
+            N/A
+    """
+    VMAX = vehicle.design_mach_number
+    FMXTOT = vehicle.mass_properties.max_zero_fuel / Units.lbs
+    WFSYS = 1.07 * FMXTOT ** 0.58 * NENG ** 0.43 * VMAX ** 0.34
+    return WFSYS * Units.lbs
+
+
+def engine_FLOPS(vehicle, prop):
+    """ Calculates the dry engine weight based on the FLOPS method
+        Assumptions:
+
+        Source:
+            The Flight Optimization System Weight Estimation Method
+
+        Inputs:
+            vehicle - data dictionary with vehicle properties                   [dimensionless]
+                -.systems.accessories: type of aircraft (short-range, commuter
+                                                        medium-range, long-range,
+                                                        sst, cargo)
+            prop    - data dictionary for the specific propulsor that is being estimated [dimensionless]
+                -.sealevel_static_thrust: sealevel static thrust of engine  [N]
+
+        Outputs:
+            WENG: dry engine weight                                         [kg]
+
+        Properties Used:
+            N/A
+    """
+    EEXP = 1.15
+    EINL = 1
+    ENOZ = 1
+    THRSO = prop.sealevel_static_thrust * 1 / Units.lbf
+    THRUST = THRSO
```

Comment:
The equations here allow scaling between a baseline engine and a scaled version. The function as written assumes the baseline engine since these are set as equal.